test/peripheral/522_borrowProxy_signatures.ts

```diff
@@ -89,20 +89,22 @@ contract('BorrowProxy - Signatures', async (accounts) => {
           await pool.mint(user1, user1, daiReserves, { from: user1 })
 
           // Post some more weth to the controller
-          await proxy.post(user1, { from: user1, value: bnify(wethTokens1).mul(2).toString() })
+          await borrowProxy.post(user1, { from: user1, value: bnify(wethTokens1).mul(2).toString() })
 
           // Give some fyDai to user1
           await fyDai1.mint(user1, fyDaiTokens1, { from: owner })
+
+          await pool.sellFYDai(user1, user1, fyDaiTokens1.div(10), { from: user1 })
```

Comment:
Is this additional sell needed? If it's just for the new repay test, let's move it there?

Reply:
Removed